packages/babel-plugin-flow-to-typescript/test/visitors/ClassDeclaration.test.ts

```diff
@@ -56,6 +56,27 @@ test('Extending class declaration types are transformed', () => {
   );
 });
 
+test('Classes with decorators are transformed', () => {
+  const result = testTransform(
+    `
+    @injectIntel
+    export class C extends React.Component<?string, ?(string | boolean)> {}
+    `
+  );
+  expect(result.babel).toMatchInlineSnapshot(
+    `
+    "export @injectIntel
```

Comment:
@zxbodya This does not seem to be correct, right. What else needs to be modified to make this work?

Reply:
[it’s configurable i think](https://babeljs.io/docs/en/babel-plugin-proposal-decorators#decoratorsbeforeexport). might need to be detected and conditionally enabled/disabled?

Reply:
indeed - config option for babel generator works - pushed a commit here

also it looks typescript supports only option with decorators before export, so I guess it make sense to add that config